data/mainnet/addresses.json

```diff
@@ -2491,5 +2495,140 @@
 				"MasterChef": "0x20ec0d06f447d550fc6edee42121bc8c1817b97d"
 			}
 		}
-	}
+	},
+	"ARBITRUM_ONE": {
+		"Storage": "0xc1234a98617385D1a4b87274465375409f7E248f",
+		"iFARM": "0x9dCA587dc65AC0a043828B0acd946d71eb8D46c1",
+		"Crv2Pool_Strategy": {
```

Comment:
I'd like to keep naming consistent with our other stuff. So name this `Curve_2pool_arbitrum`, just like in `tokens.js` and `pools.js`. Same for all other entries